assets/js/script.js

```diff
@@ -0,0 +1,73 @@
+const hum = document.querySelector('.humbarger');
+const navMenu = document.querySelector('.menu');
+
+hum.addEventListener('click', () => {
+  hum.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.menu-item').forEach((n) => n.addEventListener('click', () => {
+  hum.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
+
+const team = [
+  {
+    name: 'Iqbal Elham',
+    field: 'Full stack Developer',
+    descriptions: `Curiosity and passion for science, technology, and computer helped me to develop very well in these areas.
+    I am interested both in back-end and front-end technologies and knowledgeable in HTML, CSS, JS, and Python.`,
+    image: './assets/images/me.jpeg',
+  },
+  {
+    name: 'Andres Raul',
+    field: 'Graphic Designer',
+    descriptions: `15+ years of experience in software industry providing- Account Management and Strong experience with Internet 
+    Marketing Services selling like SEO, CMS, SEM, ORM(Web applications, on-page-optimization, off-page-optimization, keyword-research).`,
+    image: './assets/images/pic-2.png',
+  },
+  {
+    name: 'Luis Raul Zubia Maciel',
+    field: 'Full stack Developer',
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you add the **More** button feature on the desktop mobile version so you can see more or fewer(less) speakers according to the user's choice.
![Screenshot from 2022-11-24 00-25-55](https://user-images.githubusercontent.com/83042742/203656946-8b809b0c-27d2-46dd-8d65-f90ef9860e1b.png)